### DISCUSSION TITLE
Metrics-RelativePermeability

### DISCUSSION THREAD

**Discussion by HFenice** [PMEAL/OpenPNM#2017] [Q&A] [answered]
Hi,

I computed permeability by using the algorithm StokesFlow, but I've just seen that also in the section metrics a RelativePermeability module exists. What's the difference? How can I use this module? I don't find out any examples about it.

Thank you in advance.

**Comment by jgostick** [comment, marked as answer]
The relative permeability metric does an invasion percolation then computes the permeability as a function of saturation. It generates a the ubiquitous Kr = s^a curve.
